src/components/cardbuilder/card.css

```diff
@@ -364,7 +365,7 @@ button::-moz-focus-inner {
 }
 
 .cardImageIcon {
-    font-size: 5em;
+    font-size: 5em !important;
```

Comment:
Could we increase the specificity of the selector here instead of using `!important`?